### DISCUSSION TITLE
Welcome to master-thesis Discussions!

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edzob** [comment]
Nice.. deze feature kende ik nog niet :)

**Reply by JRBliekendaal** [reply]
Ik wel.. Werk veel met GitHub maar zakelijk. meestal zijn de repo's en de features er al. Dus was even zoeken waar het zat :D Op deze manier hoeft niet alles een issue te zijn
